spec/system/purchase_system_spec.rb

```diff
@@ -24,6 +26,17 @@
       page.refresh
       expect(page).to have_text("Purchased Date")
       expect(page).to have_text(1.week.ago.strftime("%Y-%m-%d"))
+      expect(page).to have_text(@total_value_all_purchases)
+    end
+
+    it "User sees total purchases value" do
+      purchase1 = create(:purchase)
```

Comment:
This spec kind of assumes it's own conclusion. It would be great if we could make the expectations more explicit.

Reply:
I've improved it slightly, please check.

Reply:
Looks awesome now :)